apps/admin/backend/schema.sql

```diff
@@ -6,6 +6,16 @@ create table elections (
   deleted_at timestamp
 );
 
+create table precincts(
```

Comment:
glad we're breaking some of these pieces out of the json blob - always wondered if there was anything stopping us from doing so

Reply:
I thought you'd like it!